src/nnue/nnue_feature_transformer.h

```diff
@@ -146,6 +147,70 @@ using psqt_vec_t = int32x4_t;
 #endif
 
 
+class Packing {
```

Comment:
If it stays like this, i.e. just a namespace for static member functions, I suggest deleting the destructor